activity/activity.go

```diff
@@ -41,6 +41,8 @@ type (
 
 	// RegisterOptions consists of options for registering an activity
 	RegisterOptions = internal.RegisterActivityOptions
+
+	ActivityExtra = internal.ActivityExtra
```

Comment:
While we don't have to, should probably wait on the name of the proto from https://github.com/temporalio/api/pull/334 and name this similarly. I'm thinking:

```suggestion
	ExtraAttributes = internal.ActivityExtraAttributes
```

But we'll see what that message gets named in that PR. I have an outstanding comment at https://github.com/temporalio/api/pull/334/files#r1422999174. I don't think it's clear to call it `ActivityRequests` there, and no reason not to use similar naming there and here since this will be applied across SDKs as a new Temporal term.